internal/testutils/provider.go

```diff
@@ -185,6 +185,8 @@ func CheckCloudInstanceTestsEnabled(t *testing.T) {
 		"GRAFANA_AUTH",
 		"GRAFANA_SM_ACCESS_TOKEN",
 		"GRAFANA_ONCALL_ACCESS_TOKEN",
+		// "GRAFANA_CLOUD_PROVIDER_URL",
+		// "GRAFANA_CLOUD_PROVIDER_ACCESS_TOKEN",
```

Comment:
Any reason this is added but commented out?

Reply:
I can add a TODO or remove these lines for now., they will need to be uncommented or added eventually. If we do that now, then the cloud-based acceptance test suite will expect a real URL and Access token to be configured in order for all of the tests gated by `CheckCloudInstanceTestsEnabled` to run, which impacts other services e.g. Synthetic Monitoring.

I guess as long as we're on a feature branch and not merging this to main, it's fine to even have them uncommented, but to be on the safe side, I wanted to have this fully dealt with by https://github.com/grafana/grafana-aws-app/issues/381

Reply:
If it's just a change for the feature branch time, +1 on leaving a TODO mentioning the issue

Reply:
A todo is totally fine, just seemed odd to include code in a feature branch PR that was commented out with no context.

Reply:
thanks, I've updated this